readme.md

```diff
@@ -255,6 +255,24 @@ To open a command prompt as administrator in the repository's directory, do the
 
 You can also refer to the "Third-Party Hosts Managers" section for further recommended solutions from third parties.
 
+## Updating hosts file on Windows using PowerShell (minimum version 3)((doesn´t require Python))
+
+**Windows 10**: Right click Start Button -> click on 'Windows PowerShell (Administrator)'
```

Comment:
I would also suggest adding instructions for Windows 7 if possible (I doubt they should be super different though).  It maybe an oldie, but it's still a goodie. 😄